parm/use_cases/model_applications/convection_allowing_models/Point2Grid_ObsOnly_obsLSR_PracticallyPerfect.conf

```diff
@@ -0,0 +1,164 @@
+POINT_2_GRID_TO_GRID = G211 
+
+# Value to pass with the -field string. See MET User's Guide for more information
+# FIELD and LEVEL both end up in the -field string
+POINT_2_GRID_INPUT_FIELD =Fscale
+POINT_2_GRID_INPUT_LEVEL =
+
+# Value to pass with the -qc argument
+POINT_2_GRID_QC_FLAGS = 0
+
+# Value to pass with the -adp argument - This is a file name with GOES Aerosol Detection Product data 
+POINT_2_GRID_ADP =
+
+# Value to pass with the -method argumen - Default is UW_MEAN, other examples are
+POINT_2_GRID_REGRID_METHOD = MAX 
+
+# Value to pass with the -gaussian-dx argument - Distance interval for gaussian smoothing
+# Default is 81.271
+POINT_2_GRID_GAUSSIAN_DX = 81.271
+
+# Value to pass with the -gaussian-radius argument - radius of influence for the gaussian smoothing
+# Default is 120
+POINT_2_GRID_GAUSSIAN_RADIUS = 120 
+
+# Value to pass with the -prob_cat_thresh argument - threshold for probability of occurrence
+POINT_2_GRID_PROB_CAT_THRESH =
+
+# Value to pass with the -vld_thresh argument - threshold for percentage of valid data .5 default 
+POINT_2_GRID_VLD_THRESH =
+
+
+# Regrid Data Plane
+OBS_REGRID_DATA_PLANE_RUN = True
+
+REGRID_DATA_PLANE_ONCE_PER_FIELD = False
+
+# Name of input field to process
+OBS_REGRID_DATA_PLANE_VAR1_INPUT_FIELD_NAME = Fscale_mask
+
+# Level of input field to process
+OBS_REGRID_DATA_PLANE_VAR1_INPUT_LEVEL = "(*,*)"
+
+# Name of output field to create
+OBS_REGRID_DATA_PLANE_VAR1_OUTPUT_FIELD_NAME = PP_probs
+
+# Mask to use for regridding
+REGRID_DATA_PLANE_VERIF_GRID = G211
+
+# Method to run regrid_data_plane, not setting this will default to NEAREST
+REGRID_DATA_PLANE_METHOD = MAXGAUSS
+
+# Regridding width used in regrid_data_plane, not setting this will default to 1
+REGRID_DATA_PLANE_WIDTH = 1
+
+# Set Gaussian dx value to add as command line argument - not added if unset or blank
+REGRID_DATA_PLANE_GAUSSIAN_DX = 81.271
+
+# Set Gaussian filter radius value to add as command line argument - not added if unset or blank
+REGRID_DATA_PLANE_GAUSSIAN_RADIUS = 120 
+
+# End of [config] section and start of [dir] section
+[dir]
+
+# Input/Output directories can be left empty if the corresponding template contains the full path to the files
+ASCII2NC_INPUT_DIR = {INPUT_BASE}/model_applications/convection_allowing_models/practically_perfect/ 
+POINT_2_GRID_INPUT_DIR = {OUTPUT_BASE}/model_applications/convection_allowing_models/practically_perfect/ 
+POINT_2_GRID_OUTPUT_DIR = {OUTPUT_BASE}/model_applications/convection_allowing_models/practically_perfect/ 
+OBS_REGRID_DATA_PLANE_INPUT_DIR = {POINT_2_GRID_OUTPUT_DIR}
+OBS_REGRID_DATA_PLANE_OUTPUT_DIR = {POINT_2_GRID_OUTPUT_DIR}
+
+# End of [dir] section and start of [filename_templates] section
+[filename_templates]
+
+# Template to look for input to ASCII2NC input storm reports 
+ASCII2NC_INPUT_TEMPLATE = "read_ascii_storm.py {INPUT_BASE}/model_applications/convection_allowing_models/practically_perfect/200205_rpts_filtered.csv"
```

Comment:
How is this script found? You may need to add the path (relative to {PARM_BASE}?) so it can be found by any user.